### PR TITLE
Edit requirements and main_analysis

### PR DESCRIPTION
**Pull request by JoaopauloRM700**
The specified versions of the libraries mentioned in the requirements document have been removed due to numerous compatibility errors, including Python. This modification allows the libraries to auto-adjust according to their dependencies, enabling them to run on newer versions of Python (e.g. 3.10.9 in my case).

In the case of the main_analysis modification, the path to locate the Analysis class was updated and the 'ANN' was removed from the columns_metrics as it was causing issues in generating plot documents.

### CODE REVIEW

File: main_analysis.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import pandas as pd
 from tabulate import tabulate
 from pathlib import Path
-from analysis_EMSE import Analisys
+from coleman4hcs.statistics.analysis import Analisys
```
Comment (jacksonpradolima) [inline main_analysis.py:11]:
```suggestion
from statistics.analysis import Analisys 

```

File: requirements.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-pingouin==0.2.2
-typing==3.6.6
-tabulate==0.8.2
-matplotlib==3.0.2
-rpy2==2.9.5
-scipy==1.1.0
-tzlocal==1.5.1
-roses==0.3.9
\ No newline at end of file
+numpy
```
Comment (jacksonpradolima) [inline requirements.txt:1]: Can you run `pip freeze` and check the versions? then. update here

**Review by jacksonpradolima — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (JoaopauloRM700) [reply requirements.txt:1]: This is the information generated when running `pip freeze`:

cffi==1.15.1
charset-normalizer==3.1.0
contourpy==1.0.7
cycler==0.11.0
fonttools==4.39.3
idna==3.4
Jinja2==3.1.2
joblib==1.2.0
kiwisolver==1.4.4
lazy_loader==0.2
littleutils==0.2.2
MarkupSafe==2.1.2
**matplotlib==3.7.1**
mkl-fft==1.3.1
mkl-random @ file:///home/builder/ci_310/mkl_random_1641843545607/work
mkl-service==2.4.0
**numpy @ file:///croot/numpy_and_numpy_base_1672336185480/work**
outdated==0.2.2
packaging==23.0
**pandas==1.5.3**
pandas-flavor==0.5.0
patsy==0.5.3
Pillow==9.5.0
**pingouin==0.5.3**
pycparser==2.21
pyparsing==3.0.9
python-dateutil==2.8.2
pytz==2023.3
requests==2.28.2
**roses==0.3.9**
**rpy2==3.4.2**
scikit-learn==1.2.2
**scipy==1.10.1**
**seaborn==0.12.2**
six @ file:///tmp/build/80754af9/six_1644875935023/work
statsmodels==0.13.5
**tabulate==0.9.0**
threadpoolctl==3.1.0
**67y45**
**tzlocal==1.5.1**
urllib3==1.26.15
xarray==2023.3.0

Comment (jacksonpradolima) [reply requirements.txt:1]: So, update the `requirements.txt` with the right values. You don't need to add the other packages

**Review by jacksonpradolima — CHANGES_REQUESTED** [review]

**Review by jacksonpradolima — APPROVED** [review]

**Pull request merged**